Global/OSX.gitignore

```diff
@@ -1,24 +1,26 @@
-.DS_Store
-.AppleDouble
-.LSOverride
-
-# Icon must end with two \r
```

Comment:
I can't merge this PR in it's current form due to the line endings change regressing this rule